.goreleaser.yaml

```diff
@@ -29,6 +29,30 @@ builds:
       - amd64
       - arm64
 
+source:
+  enabled: true
+
+sboms:
+  - artifacts: binary
+  - id: source
+    artifacts: source
+
+checksum:
+  name_template: 'checksums.txt'
+
+signs:
+  - cmd: cosign
```

Comment:
Great to have cosign integrated :) Could you maybe add sth to the installation docs about that and how to use it, or maybe just link to the cosign docs where it's discussed?
Would be great since I think it's a really good addition to the binaries in the release but wouldn't expect anyone to instantly use/ understand how it's done :)

Reply:
Ok, np will do